alea/submitters/local.py

```diff
@@ -187,15 +191,22 @@ def submit(
                 )
 
             # update poi according to poi_expectation
-            runner_args["statistical_model_args"].pop("limit_threshold", None)
-            runner = Runner(**runner_args)
-            expectation_values = runner.model.get_expectation_values(
-                **{**nominal_values, **generate_values}
-            )
-            # in some rare cases the poi is not a rate multiplier
-            # then the poi_expectation is not in the nominal_expectation_values
-            component = self.poi.replace("_rate_multiplier", "")
-            poi_expectation = expectation_values.get(component, None)
+            if runner.input_poi_expectation:
```

Comment:
Why not just "if poi_expectation" in generate_values?

Reply:
Should also be fine. What is the benefit?